projects/plugins/jetpack/class.jetpack-cli.php

```diff
@@ -897,13 +897,16 @@ public function sync( $args, $assoc_args ) {
 				break;
 			case 'start':
 				if ( ! Actions::sync_allowed() ) {
-					if ( ! Settings::get_setting( 'disable' ) ) {
+					if ( Settings::get_setting( 'disable' ) ) {
```

Comment:
haven't looked at cli for a long time. did you flip this flag from your testing? as it was always giving error message

Reply:
`wp jetpack sync enable` => this will make 'disable' 0
`wp jetpack sync disable` => this will make 'disable' 1

Then on `start` we are checking if sync actions are allowed. If not, we are looking at the 'disable' setting. 
If the disable setting is **not** set or 0, this means that sync is enabled. If the 'disable' returns `1` then it makes sense to display the corresponding message to enable sync.

I've tried in the current branch with:
`wp jetpack sync disable`
`wp jetpack sync start`
=> Gives me the correct error message ('Jetpack sync is not currently allowed for this site. It is currently disabled. Run `wp jetpack sync enable` to enable it.')

Then
`wp jetpack sync enable`
`wp jetpack sync start`
=> Gives me the "offline mode" error message (testing locally) which makes sense.

On the master branch the exact opposite happens.
I guess I could create a separate PR for this, it was just there while I was testing the cli related changes and took the opportunity to fix it in this one.

Reply:
:thumbsup: Thanks for those details, curious why I haven't stumbled over this before.